AppiumSwiftClient.xcodeproj/project.pbxproj

```diff
@@ -914,7 +910,7 @@
 				OTHER_LDFLAGS = "-ObjC";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
👍 
I'm fine to bump swift version up.
Appium server supports Xcode 10 (I mean Swift 4.2), but no reason to support the same version